.eslintrc.js

```diff
@@ -0,0 +1,55 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "jest": true,
+        "mocha": true,
+        "node": true,
+        "es6": true
+    },
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "project": "tsconfig.json",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    extends: [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:react/recommended"
+    ],
+    "rules": {
+        "@typescript-eslint/class-name-casing": "error",
+        "@typescript-eslint/indent": "off",
```

Comment:
Maybe we should use indentation 2? It is very readable, but this is just my opinion.